packages/localize/src/localize.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+/// <reference path="../localize.d.ts" />
+
+/**
+ * Localize a template literal string.
+ *
+ * This implementation of the global `$localize` tag simply returns the original template with
+ * the interpolations evaluated - effectively an identity for the original template literal.
+ *
+ * Generally this version is used in development or where there is no need to transform the
+ * localized template literals.
+ *
+ * @param messageParts a collection of the static parts of the template string.
+ * @param expressions a collection of the values of each placeholder in the template string.
+ * @returns the reconstituted string, with the `messageParts` and `expressions` interleaved
+ * together.
+ */
+
+(function(global: any) {
+  const $localize: Localize = function localize(): string {
+    const args = $localize.translate ? $localize.translate(arguments as any) :
+                                       Array.prototype.slice.apply(arguments);
```

Comment:
```suggestion
                                       Array.prototype.slice.apply(arguments, 1);
```
So that we can skip the first arg

Reply:
This is a conscious choice. We must use the first item from the `messageParts` array before the loop, which means that we must loop over `messageParts[1...]`, so it makes sense to use `slice()` to make the `expressions` one-based as well.

If we use `slice(1)` then the two arrays are offset by one from each other, and we would need to compute `i-1` for each expression.